### PR TITLE
Release/mip9.0.4

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- increases memory allocation for samtools_subsample_mt to 4G
- Updates the check of vep plugin paths to the new config format
- Adds a check to see that all vep plugins are installed 
- Cd into temp dir before executing cadd in order to escape snakemake locks

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by jemten** [comment]
Testing on prokitten which has previous fail on samtools_subsample_mt due to memory error  
analysis starts ✅ 
image

File: lib/MIP/Vep.pm [hunk 1]
```diff
@@ -187,16 +187,18 @@ sub check_vep_custom_annotation {
 
 sub check_vep_plugin {
 
-## Function : Check VEP plugin options
+## Function : Check VEP plugins and ptions
```
Comment (henrikstranneheim) [inline lib/MIP/Vep.pm:190]:
```suggestion
## Function : Check VEP plugins and options
```

File: t/parse_vep_plugin.t [hunk 2]
```diff
@@ -62,21 +64,45 @@ diag(   q{Test parse_vep_plugin from Active_parameter.pm v}
...
+#my %active_parameter_href = (
+#    vep_plugin => {
+#        dbNSFP => {
+#            exists_check =>
+#                q{directory},
+#            path         => cwd(),
+#            parameters   => [qw{ param_1 param_2 }],
+#        },
+#    },
+#);
```
Comment (henrikstranneheim) [inline t/parse_vep_plugin.t:100]:
```suggestion
```

**Review by henrikstranneheim — APPROVED** [review]
Looks good!

File: t/parse_vep_plugin.t (hunk 2 above)
Comment (jemten) [reply t/parse_vep_plugin.t:100]: I knew I forgot something

**Comment by jemten** [comment]
@ramprasadn, could you take a look at this Ram. I'm trying to patch production and it would be nice with a review

**Review by ramprasadn — COMMENTED** [review]
Looks good!

**Pull request merged**